chia/full_node/full_node_api.py

```diff
@@ -1272,7 +1272,7 @@ async def send_transaction(
         await self.full_node.transaction_queue.put(queue_entry, peer_id=None, high_priority=True)
         try:
             with anyio.fail_after(delay=45):
-                status, error = await queue_entry.done
+                status, error = await queue_entry.done.wait()
```

Comment:
wouldn't it be simpler (and maybe more robust) to just use:
```
asyncio.wait_for(queue_entry, 45)
```

instead of the `fail_after()` context manager?

Reply:
Using `asyncio.wait_for()` wouldn't be terrible, but I don't think it is better.  Using `anyio.fail_after()` has a few benefits in my mind.

1) one pattern that can handle more cases
2) no callbacks be passed around (they can be useful, but not my preferred pattern when easily avoided)
3) in general, i hope to shift away from asyncio and towards anyio for most things due to them leading towards better patterns in general.  if we can consistently use anyio that will make it more clear what people should use in new code.

Now sure, transitions take time and there may be cases where we continue to use asyncio directly, but I think they should have distinct valuable reasons.